### PR TITLE
Remove reference to scheduled Whitehall publications

### PR DESCRIPTION
**Pull request by ChrisBAshton**
This was ported to the new platform in https://github.com/alphagov/govuk-helm-charts/pull/1677

### CODE REVIEW

**Comment by sengi** [comment]
The old "not queued" alert is `Edition.future_scheduled_editions.count - ScheduledPublishingWorker.queue_size > 0`. I think that can only be true if the "overdue" alert is also firing (i.e. `Edition.due_for_publication.count > 0`). Is that a valid statement?

**Comment by samsimpson1** [comment]
I'm not aware of any discussions around these checks, but the two metrics are actually different on staging and integration, (in staging, the overdue count is 32 and the unenqueued count is over 500)

File: views/blinken.erb [hunk 1]
```diff
@@ -15,7 +15,7 @@
         You need to be on the VPN to see the Icinga alerts below. You can ignore most alerts - the only ones that still need attention are:
       </p>
       <ul>
-          <li>“scheduled publications in Whitehall not queued” / “overdue publications in Whitehall”
+          <li>“overdue publications in Whitehall”</li>
```
Comment (samsimpson1) [inline views/blinken.erb:18]: This entire line can be removed now, since this has been merged in https://github.com/alphagov/govuk-helm-charts/pull/1722

**Comment by sengi** [comment]
Thanks @samsimpson1!

**Pull request merged**